### PR TITLE
Update almost all npm deps and introduce webpack

### PR DESCRIPTION
**Pull request by MetalMatze**
Webpack is awesome for developing with react.
There's a `webpack -w` command which I've introduced as `make watch`.
It caches all modules and only recompiles the ones, that changed.
Building after changing a react component now takes ms instead of 5s.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,46 +2,46 @@
...
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/drone/drone-ui.git"
   },
   "author": "",
-  "license": "ISC",
+  "license": "Apache",
```
Comment (tboerger) [inline package.json]: I think that needs to be Apache-2.0

**Comment by tboerger** [comment]
One tiny NITPICK, otherwise LGTM

File: package.json (hunk 1 above)
Comment (MetalMatze) [inline package.json]: Thanks, that was just a quick fix I did without proper research. It's updated in this PR.

**Comment by tboerger** [comment]
Still LGTM, but I don't have the permissions to merge :)

**Comment by MetalMatze** [comment]
Added less support for webpack. I just hurts to run `make less` all the time. I want this to be automatic.

**Comment by tboerger** [comment]
I dislike less in general :)

**Comment by MetalMatze** [comment]
I like sass more! But without it's still a pain...

**Pull request closed without merging**